atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingKeyValueService.java

```diff
@@ -302,11 +297,7 @@ public RowColumnRangeIterator getRowsColumnRange(TableReference tableRef,
             ColumnRangeSelection columnRangeSelection,
             int cellBatchHint,
             long timestamp) {
-        //noinspection unused - try-with-resources closes trace
-        try (CloseableTrace trace = startLocalTrace("getRowsColumnRange({}, {} rows, {} hint, ts {})",
-                tableRef, Iterables.size(rows), cellBatchHint, timestamp)) {
-            return delegate().getRowsColumnRange(tableRef, rows, columnRangeSelection, cellBatchHint, timestamp);
-        }
+        return delegate().getRowsColumnRange(tableRef, rows, columnRangeSelection, cellBatchHint, timestamp);
```

Comment:
Probably worth a comment here why we’re not tracing due to span not capturing full unit of work due to lazy iterator